includes/repositories/EUtilsAssemblyRepository.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+class EUtilsAssemblyRepository extends EUtilsRepositoryInterface {
+
+  /**
+   * Required attributes when using the create method.
+   *
+   * @var array
+   */
+  protected $required_fields = [
+
+  ];
+
+  /**
+   * Cache of data per run.
+   *
+   * @var array
+   */
+  protected static $cache = [
```

Comment:
- should we standardize the cache keys?
- can the cache live in the general class?